server/code/setup/cpdb

```diff
@@ -101,10 +101,10 @@ class DbSetup(object):
         # Add in output level. By default, Liquibase defaults to "off"
         liquibase_options += " --logLevel=%s" % ('debug' if self.verbose else 'severe')
 
-        print liquibase_options
+        print(liquibase_options)
```

Comment:
please add `from __future__ import print_function` at the very beginning of this file for best Py2 compatibility.